pkg/internal/wordpress/pod_template.go

```diff
@@ -347,6 +397,32 @@ func (wp *Wordpress) mediaContainers() []corev1.Container {
 
 func (wp *Wordpress) initContainers() []corev1.Container {
 	containers := []corev1.Container{}
+	installCredentials := wp.installCredentials()
+
+	if installCredentials != nil {
+		scheme := "http"
+		if len(wp.Spec.TLSSecretRef) > 0 {
+			scheme = "https"
+		}
+		url := fmt.Sprintf("%s://%s/", scheme, wp.Spec.Domains[0])
+
+		installCmd := []string{
```

Comment:
I think you need a small bash script here to check that `wp core is-installed` first. I recall that `wp core install` fails if the site is already installed, leasing to pods not being started anymore.

Reply:
No, it exists with `0` and with the message that the wordpress is already installed.

Reply:
I thought to have something like ``` wp core install || true``` but is not necessary.